src/component/footer/proxy-status.js

```diff
@@ -23,7 +39,7 @@ const ProxyStatus = ({proxyStatus, proxyMessage, proxyWindow}) => {
 };
 
 ProxyStatus.propTypes = {
-  proxyStatus: oneOf(['working', 'partial', 'offline']).isRequired,
+  proxyStatus: oneOf(Object.keys(constants).map(key => constants[key])).isRequired,
```

Comment:
I think what you have there is better, so we don't need to write down the same status twice. :+1: :)